### PR TITLE
refactor(GiniBankSDK): Rename localizable strings for Skonto

### PR DESCRIPTION
**Pull request by ValentinaIancu-Gini**
PP-602

### CODE REVIEW

File: BankSDK/GiniBankSDK/Sources/GiniBankSDK/Resources/de.lproj/Localizable.strings [hunk 1]
```diff
@@ -86,22 +86,22 @@
...
-"ginibank.skonto.info.amount.title" = "Betrag nach Abzug";
-"ginibank.skonto.info.date.title" = "Ablaufdatum Skonto";
-"ginibank.skonto.notapplied.title" = "Ohne Skonto";
-"ginibank.skonto.notapplied.amount.title" = "Betrag ohne Abzug";
-"ginibank.skonto.total.amount.skonto" = "%@ Skonto";
-"ginibank.skonto.paybutton.title" = "Zahlung fortsetzen";
+"ginibank.skonto.backbutton.title" = "Zurück";
+"ginibank.skonto.withdiscount.title" = "Mit Skonto";
+"ginibank.skonto.active" = "• Aktiviert";
+"ginibank.skonto.infobanner.normal.message" = "Zahlung in %@: %@ Skonto.";
```
Comment (mrkulik) [inline BankSDK/GiniBankSDK/Sources/GiniBankSDK/Resources/de.lproj/Localizable.strings:92]: [Suggestion] @ValentinaIancu-Gini what do you think about this rename? infobanner.normal -> infobanner.default

**Review by mrkulik — CHANGES_REQUESTED** [review]
@ValentinaIancu-Gini all looks good, thank you! left just 1 suggestion about rename

File: BankSDK/GiniBankSDK/Sources/GiniBankSDK/Resources/de.lproj/Localizable.strings (hunk 1 above)
Comment (ValentinaIancu-Gini) [reply BankSDK/GiniBankSDK/Sources/GiniBankSDK/Resources/de.lproj/Localizable.strings:92]: I think is a good idea. I will talk with Mahdi to see if they want to use the same version.

**Review by mrkulik — APPROVED** [review]
@ValentinaIancu-Gini perfect, let's wait checks completion and merge it! Thank you!

**Pull request merged**